### ISSUE TITLE
B-Tax versioneer issue in current master

### ISSUE THREAD

**Issue by PeterDSteinberg** [open-source-economics/B-Tax#67]
There is an inability to compute the version.  The problem has something to do with `versioneer.py` and tagging versions or lack thereof:
```
In [1]: from btax._version import get_versions

In [2]: get_versions()
Out[2]:
{'dirty': None,
 'error': 'unable to compute version',
 'full-revisionid': None,
 'version': '0+unknown'}
```

**Comment by talumbau** [comment]
Hmm.. I'm not sure what versioneer does if there is no tag at all in the repo. You might fix this just for now by doing:

```
git tag 0.1
git push; git push --tags
```

**Comment by PeterDSteinberg** [comment]
@jdebacker Can you do the sequence of commands above @talumbau  mentions - I am getting 403 permissions errors when attempting to push `--tags`

**Comment by jdebacker** [comment]
Ok - I did this, but don't see an effect on the PR.  Do I need to specify origin?

@PeterDSteinberg

**Comment by PeterDSteinberg** [comment]
Yes @jdebacker I think the tag(s) need to be pushed to the main repo, not one of our forks.

**Comment by jdebacker** [comment]
Ok, did `git push --tags upstream`.  I think it worked - let me know.

**Comment by PeterDSteinberg** [comment]
This issue is solved - see [policybrain-builder](https://github.com/open-source-economics/policybrain-builder)
